examples/elasticsearch/blueprint-v2/elasticsearch-blueprint.yaml

```diff
@@ -26,17 +31,23 @@ actions:
         - |
           host_name="{{ .Object.spec.serviceName }}.{{ .StatefulSet.Namespace }}.svc.cluster.local"
           backup_file_path='backup.gz'
-          elasticdump --bulk=true --input=http://${host_name}:9200 --output=/backup
-          gzip /backup
-          kando location push --profile '{{ toJson .Profile }}' --path "${backup_file_path}" --output-name "kopiaOutput" /backup.gz
+          master_username="{{ index .Phases.backupToStore.Secrets.esMasterCredSecret.Data "username" | toString }}"
+          master_password="{{ index .Phases.backupToStore.Secrets.esMasterCredSecret.Data "password" | toString }}"
+          NODE_TLS_REJECT_UNAUTHORIZED=0 elasticdump --bulk=true --input=https://${master_username}:${master_password}@${host_name}:9200 --output=/backup
+          gzip -c /backup | kando location push --profile '{{ toJson .Profile }}' --path "${backup_file_path}" --output-name "kopiaOutput" -
```

Comment:
Just curious: Can we restore directly from the `/backup` directory created by `elasticdump`?
`kando` using `kopia` can handle the compression part without using `gzip`

Reply:
I can try that, I referred mongodb-blueprint and there we were already passing compressed file to kando push [https://github.com/kanisterio/kanister/blob/0a9d583f94f30aec389cd37834a2cf8c40710964/examples/mongodb/blueprint-v2/mongo-blueprint.yaml#L34](https://github.com/kanisterio/kanister/blob/0a9d583f94f30aec389cd37834a2cf8c40710964/examples/mongodb/blueprint-v2/mongo-blueprint.yaml#L34)